### ISSUE TITLE
UMD won't compile. Bad Src filename.

### ISSUE THREAD

**Issue by RavenHursT** [insin/nwb#468]
This issue is a: Bug report / Question / support request

When I try to run `npm run build` I get the following: 

![image](https://user-images.githubusercontent.com/496983/43422081-46c8bb26-940e-11e8-8bac-d6685d6db8f0.png)

I was getting the same error for the `/lib` and `/es` builds, but was able to make them pass, simply by modifying the `package.json` from:

```
"main": "lib/index.js",
"module": "es/index.js",
```
to: 

```
"main": "lib/react-flex-rating.js",
"module": "es/react-flex-rating.js",
```

Where can I tell `nwb` what file it _should_ be looking looking for?

```
node -v
v10.6.0
npm -v
6.1.0
nwb -v
v0.21.5
```

Which modules are installed in your project?

```
npm ls --depth=0
```
![image](https://user-images.githubusercontent.com/496983/43422286-db7037ae-940e-11e8-9f9a-8f0c5828b1e5.png)

I found this in the documentation: https://github.com/insin/nwb/blob/master/docs/Configuration.md#entry-string

I tried adding it to my nwb config:
```
module.exports = {
  type: 'react-component',
  npm: {
    esModules: true,
    umd: {
      global: 'ReactFlexRating',
      entry: 'src/react-flex-rating.js',
      externals: {
        react: 'React'
      }
    }
  },
  webpack: {
    html: {
      template: 'demo/src/index.html'
    }
  }
}
```

.. But now I get this error when trying to build:

![image](https://user-images.githubusercontent.com/496983/43422864-9242ff9c-9410-11e8-99dd-17525a0cf26f.png)

**Comment by RavenHursT** [comment]
FYI... changing everything back to default `index.*` works just fine.

Would like to have custom filenames, however..

**Comment by insin** [comment]
`umd.entry` config is only available from v0.23.0 onward, which isn't released yet.

Updated [the docs](https://github.com/insin/nwb/blob/master/docs/Configuration.md#entry-string) to indicate that.
